integration/oidc/authorization-code/README.md

```diff
@@ -0,0 +1,197 @@
+
+1. In Azure AD, go to the `App Registrations` section.
+1. Click on the `New application registration` button.
+1. Enter the following information:
+
+    Name: `My aspnet core Web App`
+    Application Type: `Web app / API`
+    Sign-on URL: `http://localhost:8080`
+
+1. When you are ready, click on the `Create` button.
+
+You now have registered this example application. You now need to finish
+configuring it in order to get the auhtentication working properly.
+
+1. Go to the `My aspnet core Web App` application registration page in Azure AD.
+1. Copy the Application ID and save it for later use. This is the `Client Id`
+of the application.
+1. Click on the `Settings` button.
+1. Go to the `Reply URLs` section.
+1. Add `http://localhost:8080/signin` to the list and click on the `Save`
+button.
+1. Return to the sections list and go to the `Keys` section.
+1. In the `Passwords` sub-section, add a line with the following information:
+
+    Description: `secret`
+    Expires: `Never expires`
+    Value: A random Guid (see the following note)
+
+    **About Generating a Guid**
+
+    Open a PowerShell terminal and enter the following command:
+    ```powershell
+    [guid]::NewGuid().ToString()
+    ```
+    Copy the result and enter in the value section of the form.
+
+1. Click on the `Save` button.
+1. Copy the value that has been generated after the save. It is the `Client Secret`
+of this application.
+1. Return to the sections list and go to the `Required permissions` section.
+1. Click on the `Add` button.
+1. Click on the `Select an API` button.
+1. Search for the WorkflowGen's GraphQL API application that you've configured
+prior to this guide, click on it and click on the `Select` button.
+1. In the `Select permissions` section, check all checkboxes and click on the
+`Select` button.
+1. Click on the `Done` button.
+
+You should now see WorkflowGen's GraphQL API in the list of required permissions
+for this example application.
+
+You should now have four (4) pieces of information in your possession at this
+point:
+
+1. This application registration's Application ID or `Client Id`
+1. This application registration's Password Key or `Client Secret`
+1. The App ID URI of WorkflowGen's GraphQL API. It should look like this:
+
+    `http://localhost:8888/wfgen/graphql`
+
+1. Your Azure AD Directory ID which can be found in the properties section of your
+Azure AD page.
+
+## Configure the Example Application
+The configuration of the example is straightforward and takes a lot less steps
+than configuring it in Azure AD.
+
+### Prerequisites
+1. Make sure to have installed the dotnet core sdk version 2.2. The `dotnet`
+commandline tool should be available in a terminal.
+1. Make sure that the user that you're using is a valid WorkflowGen user.
+
+Optionally, you can install Visual Studio Code to run and debug this example.
+There is existing vscode settings in the repository to make it easier to get
+started.
+
+### Get Started
```

Comment:
### Get started

^^^ this implies there's another section after this one. What are they actually doing here?